flask/app.py

```diff
@@ -0,0 +1,13 @@
+from flask import Flask, render_template, request,json
+
+app= Flask('__main__')
+
+@app.route('/')
+
+def main():
+	return render_template('index.html')
+
+
+
+if __name__ == '__main__':
+	app.run('0.0.0.0',8080)
```

Comment:
We so not need the app to have a specific port etc, It would be great If you could replace it with `app.run()` only